app/boards/shields/boardsource3x4/boardsource3x4.keymap

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+
+/ {
+    keymap {
+        compatible = "zmk,keymap";
+
+        default_layer {
+
+            bindings = <
+    &tog 1 &cp M_PREV &cp M_PLAY &cp M_NEXT
+    &mo 2  &cp M_VOLD &kp UARW   &cp M_VOLU
+    &mo 3  &kp LARW   &kp DARW   &kp RARW
+            >;
+
+        };
+
+        num_layer {
+            bindings = <
+    &tog 1       &kp NUM_7 &kp NUM_8 &kp NUM_9
```

Comment:
You should be able to make this first binding just `&trans` to be transparent to flow to the layer below.